### DISCUSSION TITLE
Backend > Customers > Online Customers - Improvements in grid

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2478] [Ideas]
Go to **Backend > Customers > Online Customers** and please take a look to the grid. Here are a few improvements to be done:

1. First **Name / Middle Name / Last Name** should be 1 column for saving space. If we check Customers > Manage Customers the grid has only one column named "Name". 
2. The **IP Address** column must allow filtering
3. On the column **Last URL** the link should be the friendly ones. In my case I get listed only links in this format ```https://www.mydomain.tld/catalog/category/view/id/157```. It is practically impossible to understand which product or category is being visited. However, there are situations when links with names appear.

**Comment by sreichel** [comment]
1. imho ... we have enough space in this grid. I would keep it
2. sorting/filtering will not work ... IP address are not stored as plain text (e.g. `0xac160006` for `172.22.0.6`)
3. would it be enough to make the URL clickable?
